src/renderers/webgl/WebGLShader.js

```diff
@@ -16,22 +16,26 @@ function addLineNumbers( string ) {
 
 }
 
-function WebGLShader( gl, type, string ) {
+function WebGLShader( gl, type, string, check ) {
 
 	var shader = gl.createShader( type );
 
 	gl.shaderSource( shader, string );
 	gl.compileShader( shader );
 
-	if ( gl.getShaderParameter( shader, gl.COMPILE_STATUS ) === false ) {
+	if(check === true) {
 
-		console.error( 'THREE.WebGLShader: Shader couldn\'t compile.' );
+		if (gl.getShaderParameter(shader, gl.COMPILE_STATUS) === false) {
```

Comment:
Code style -- should have spaces inside of parentheses.

Reply:
done